lib/middleman-livereload/reactor.rb

```diff
@@ -40,11 +40,25 @@ def reload_browser(paths = [])
         end
       end
 
+      def websocket_options
+        @websocket_options ||= begin
+          return {} unless @options[:ssl_certificate] || @options[:ssl_private_key]
+          {
+            secure: true,
+            tls_options: {
+              private_key_file: @options[:ssl_private_key],
+              cert_chain_file: @options[:ssl_certificate]
+            }
+          }
+        end
+      end
+
```

Comment:
How about something like this?

```ruby
def secure_websockets?
  @options[:wss_certificate] && @options[:wss_private_key]
end
```

This method `secure_websockets?` can then be reused in both `websockets_options` and for the logging.